core/src/main/java/org/pentaho/di/connections/vfs/VFSConnectionProvider.java

```diff
@@ -57,4 +57,37 @@ default FileObject getDirectFile( T connectionDetails, String path ) throws Kett
     return KettleVFS.getFileObject( pvfsUrl, new Variables(), getOpts( connectionDetails ) );
   }
 
+
+  /**
+   * Test the VFS connection with provided connection details .
+   *
+   *
+   * @return {@code true} if the provided rootPath is valid
+   */
+  default boolean test( T connectionDetails, VFSConnectionTestOptions connectionTestOptions ) throws KettleException {
+    return test( connectionDetails );
+  }
+
+  /**
+   * Method to resolve root path for a given VFS connection.
```

Comment:
There's no need to say it is a method. You can go directly to what it does. E.g. `Gets the resolved root path of a given connection.`

Also, `@return The non-empty resolved root path, if any; {@code null}, if. none.`